apiary.apib

```diff
@@ -954,11 +954,568 @@ Outputs the list of peers that were removed.
+        Transfer-Encoding: chunked
+        Date: Tue, 02 Feb 2016 17:04:39 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes (Error)
+        + Message: "Failed to get config value:  key has no attributes"
+        + Code: 0
+
+    + Body
+
+        ```
+        {
+          "Message": "Failed to get config value:  key has no attributes",
+          "Code": 0
+        }
+        ```
+
++ Request With Argument
+
+    #### curl
+
+        curl -i http://localhost:5001/api/v0/config?arg=API.HTTPHeaders
+
+    + Body
+
+        ```
+        curl -i http://localhost:5001/api/v0/config?arg=API.HTTPHeaders
+        ```
+
++ Response 200
+
+    + Headers
+
+        ```
+        Access-Control-Allow-Headers: X-Stream-Ouptut, X-Chunked-Output
+        Access-Control-Expose-Headers: X-Stream-Ouptut, X-Chunked-Output
+        Content-Type: application/json
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        Date: Tue, 02 Feb 2016 16:59:11 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes (object)
+        + `Key`: "Datastore" (string)
+        + `Value`: null (object, nullable) - The config entry
+
+    + Body
+
+        ```
+        {
+          "Key": "API.HTTPHeaders",
+          "Value": null
+        }
+        ```
+
++ Request As SubCommand
+
+    #### curl
+
+        curl -i http://localhost:5001/api/v0/config/API.HTTPHeaders
+
+    + Body
+
+        ```
+        curl -i http://localhost:5001/api/v0/config/API.HTTPHeaders
+        ```
+
++ Response 200
+
+    + Headers
+
+        ```
+        Access-Control-Allow-Headers: X-Stream-Ouptut, X-Chunked-Output
+        Access-Control-Expose-Headers: X-Stream-Ouptut, X-Chunked-Output
+        Content-Type: application/json
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        Date: Tue, 02 Feb 2016 16:59:11 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes (object)
+        + `Key`: "Datastore" (string)
+        + `Value`: null (object, nullable) - The config entry
+
+    + Body
+
+        ```
+        {
+          "Key": "API.HTTPHeaders",
+          "Value": null
+        }
+        ```
+
++ Request With POST
+
+    #### curl
+
+        curl -i "http://localhost:5001/api/v0/config?arg=Datastore.Path&arg=kitten"
+
+    + Body
+
+        ```
+        curl -i "http://localhost:5001/api/v0/config?arg=Datastore.Path&arg=kitten"
+        ```
+
++ Response 200
+
+    + Headers
+
+        ```
+        Content-Type: application/json
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        Date: Tue, 02 Feb 2016 22:25:00 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes (object)
+        - Key: "Datastore.Path" (string)
+        - Value: "kitten" (string)
+
+
+    + Body
+
+        ```
+        {
+          "Key": "Datastore.Path",
+          "Value": "kitten"
+        }
+        ```
+
++ Request With POST and JSON Flag With Invalid JSON Argument
```

Comment:
on top it says it is accessible with `GET`, here says that is POST, although the `curl` lines show a GET request too (this should definitely be a POST request since it is causing changes to the IPFS Repo)